File: src/main/java/com/fendyk/commands/ConfirmCommands.java
package com.fendyk.commands;

import com.fendyk.Main;
import com.fendyk.managers.ConfirmCommandManager;
import dev.jorel.commandapi.CommandAPICommand;

public class ConfirmCommands {

    public ConfirmCommands() {
        new CommandAPICommand("confirm")
                .executesPlayer((player, args) -> {
                    ConfirmCommandManager.confirmedCommand(player);
                }).register();
    }

}

File: src/main/java/com/fendyk/commands/ConfirmCommands.kt
package com.fendyk.commands

import com.fendyk.managers.ConfirmCommandManager
import dev.jorel.commandapi.CommandAPICommand
import dev.jorel.commandapi.executors.PlayerCommandExecutor
import org.bukkit.entity.Player

class ConfirmCommands {
    init {
        CommandAPICommand("confirm")
            .executesPlayer(PlayerCommandExecutor { player: Player, args: Array<Any?>? ->
                ConfirmCommandManager.confirmedCommand(
                    player
                )
            }).register()
    }
}
